webapp/src/main/java/life/qbic/views/register/UserRegistrationHandler.java

```diff
@@ -57,30 +60,16 @@ private void addListener() {
         });
   }
 
-  private void setEmptyFieldsInvalid() {
-    if (userRegistrationLayout.password.isEmpty()) {
-      userRegistrationLayout.password.setInvalid(true);
-    }
-    if (userRegistrationLayout.fullName.isEmpty()) {
-      userRegistrationLayout.fullName.setInvalid(true);
-    }
-    if (userRegistrationLayout.email.isEmpty()) {
-      userRegistrationLayout.email.setInvalid(true);
-    }
-  }
-
-  private void handleUserException(String reason) {
-    if (reason.equalsIgnoreCase("Password shorter than 8 characters.")) {
-      userRegistrationLayout.passwordTooShortMessage.setVisible(true);
-    }
-    if (reason.equalsIgnoreCase("User with email address already exists.")) {
-      userRegistrationLayout.alreadyUsedEmailMessage.setVisible(true);
-    }
-  }
-
   private void resetErrorMessages() {
-    userRegistrationLayout.passwordTooShortMessage.setVisible(false);
     userRegistrationLayout.alreadyUsedEmailMessage.setVisible(false);
+    userRegistrationLayout.errorMessage.setVisible(false);
+    userRegistrationLayout.passwordTooShortMessage.setVisible(false);
+  }
+
+  private void resetComponentErrors() {
+    userRegistrationLayout.fullName.setErrorMessage("A username must be at least 1 character");
+    userRegistrationLayout.email.setErrorMessage("Please provide a valid email");
+    userRegistrationLayout.password.setErrorMessage("Password too short");
```

Comment:
I am confused, this reads as if you set the error rather than reset

Reply:
Good Point i renamed the method to make it's purpose more clear 👍